packages/itwin/tree-widget/src/components/TreeWidgetUiItemsProvider.tsx

```diff
@@ -51,9 +51,9 @@ export class TreeWidgetUiItemsProvider implements UiItemsProvider {
       const trees: SelectableContentDefinition[] = [];
 
       if (!this._treeWidgetOptions?.hideTrees?.modelsTree) {
-        trees.push({
-          label: TreeWidget.translate("modeltree"),
-          id: "model-tree",
+        const modelTree = {
+          label: TreeWidget.translate("modelstree"),
+          id: "models-tree",
```

Comment:
lets extract these ids we've hardcoded for each of the trees and put them into a constant that we export in the types.ts file. then we can just use those constants here, and others can use those as well

Reply:
I have created the three constants for all three hardcoded treeIds in type.ts